client/Index.jsx

```diff
@@ -4,24 +4,27 @@ import MainDescription from './components/MainDescription.jsx';
 class Index extends React.Component {
   constructor(props) {
     super(props);
-    this.state = { description: null };
+    this.state = { descriptions: [] };
   }
 
   componentDidMount() {
     axios.get('/api/description/0').then(({ data }) => {
-      this.setState({ description: data }).catch((error) => {
-        console.log('Error fetching data', error);
+      this.setState({ descriptions: data.description }, () => {
+        console.log('Updated state,', this.state);
       });
     });
+    // });
   }
```

Comment:
dont forget to handle errors in the catch

Reply:
Ugh. I had the catch there and deleted it! Thanks for catching it.